org.uqbar.project.wollok.lib/src/wollok/lang/WSet.xtend

```diff
@@ -23,8 +23,9 @@ class WSet extends WCollection<Set<WollokObject>> {
 	}
 	
 	override protected def verifyWollokElementsContained(Collection set, Collection set2) {
-		set.forall [ elem |
-			wrapped.exists[ it.wollokEquals(elem) ]
+		set2.forall [ elem |
```

Comment:
This implementation is not good enough, we should check for the size of the sets.